build/images/Dockerfile.build.windows

```diff
@@ -0,0 +1,46 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2019 as cni-binaries-windows
+SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+ENV CNI_PLUGINS="./host-local.exe"
+
+RUN mkdir -Force C:\opt\cni\bin
+
+RUN curl.exe -LO https://github.com/containernetworking/plugins/releases/download/v0.8.5/cni-plugins-windows-amd64-v0.8.5.tgz; \
+    tar -xzf cni-plugins-windows-amd64-v0.8.5.tgz  -C C:\opt\cni\bin ${env:CNI_PLUGINS}; \
+    rm cni-plugins-windows-amd64-v0.8.5.tgz
+
+
+FROM winamd64/golang:1.13 as antrea-build-windows
+
+WORKDIR /antrea
+
+COPY go.mod /antrea/go.mod
+
+RUN go mod download
+
+COPY . /antrea
+
+RUN make windows-bin
+
+
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+RUN NET user admin /add
+RUN NET LOCALGROUP Administrators /add admin
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="A Docker image to deploy the Antrea CNI. Requires the Antrea binaries to be built prior to building the image."
```

Comment:
The description is outdated, I change it and keep same with Docker.build.ubuntu.

@antoninbas The file `Dockerfile.build.windows` is still in process. It's because `golang:1.13` image on windows does not support `make tool`. So we cannot run `make windows-bin` to build binaries. I'm working on adding the support to `make tool` and other GNU utilities. Glad to hear from you if you have any experience or suggestion for it.

Thanks,
Rui

Reply:
looks like you figured out a solution, lgtm